pkg/apb/types.go

```diff
@@ -200,3 +201,10 @@ type RecoverStatus struct {
 	InstanceID uuid.UUID `json:"id"`
 	State      JobState  `json:"state"`
 }
+
+// ExecutionContext - Contains the information necessary to track and clean up an APB run
+type ExecutionContext struct {
```

Comment:
Not sure if it needs to be at a type level, but there's probably a better way I can do this that makes more clear the difference between the context of the service instance and the context of the apb. Open to suggestions if you have any, I'm going to think on it for a bit.